### PR TITLE
v2.1.1 release

### PR DESCRIPTION
**Pull request by wjakob**

### CODE REVIEW

**Comment by wjakob** [comment]
Not sure I understand the hash error message. For the .tar.gz downloaded from PyPi, I get:
```
 $ shasum -a 256  ~/Downloads/pybind11-2.1.1.tar.gz 
c169a384ea046a1672ac83e9aed30660133e9bad3afd82fa99b794df7b416f6d  /Users/wjakob/Downloads/pybind11-2.1.1.tar.gz
```

**Comment by dean0x7d** [comment]
> Not sure I understand the hash error message. For the .tar.gz downloaded from PyPi

PR #14 switched to the GitHub archive instead of PyPI. Different hash.

**Comment by SylvainCorlay** [comment]
Good catch.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{% set version = "2.1.0" %}
-{% set sha256 = "2860f2b8d0c9f65f0698289a161385f59d099b7ead1bf64e8993c486f2b93ee0" %}
```
Comment (SylvainCorlay) [inline recipe/meta.yaml]: Also, the build number should be reset to 0.

**Comment by SylvainCorlay** [comment]
OK, opened #18 with the hash and build number.

**Comment by wjakob** [comment]
thanks!

**Pull request closed without merging**